src/main/java/com/commercetools/project/sync/util/ChunkUtils.java

```diff
@@ -0,0 +1,100 @@
+package com.commercetools.project.sync.util;
```

Comment:
ChunkUtils and its tests, should be removed as it's added in java-sync with 3.2.0

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/234/commits/cdcb90a5c897a597149264be4c67c24a25068b65